doc/source/yt4differences.rst

```diff
@@ -92,6 +92,45 @@ arbitrary_grid = ds.arbitrary_grid([0.0, 0.0, 0.0], [5, 5, 5],dims=[10, 10, 10])
 density = arbitrary_grid[('gas', 'density')]
 print(density)
 ```
+Off-Axis Projection for SPH Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The current `off_axis_projection` function will now support SPH projection plots.
+
+The following code generates an off-axis projection of an example dataset with a given
+normal vector, resolution, center, width, and smoothing quantitiy. 
+
+.. code-block:: python
+    import yt
+
+    ds = yt.load('GadgetDiskGalaxy/snapshot_200.hdf5')
+
+    normal_vector = [0., 1., 0.]
```

Comment:
maybe make this `[1, 1, 1]` or some other vector that isn't aligned with the coordinate axes.